microsoft/PowerToys
# [Run] Add Fluent icon for PT Run

## Summary of the Pull Request

**What is this about:**
Adding Fluent icon for Launcher as an .ico, so it shows up in e.g. Task manager

**What is include in the PR:** 
Removed placeholder.ico and added Run.ico

**How does someone test / validate:** 
Run launcher, check PowerLauncher.exe in Taskmanager

![image](https://user-images.githubusercontent.com/9866362/109485002-6d6a8780-7a81-11eb-846a-e33c03f17dbb.png)


@enricogior Not sure if this asset needs to be added to the installer?


## Quality Checklist

- [X] **Linked issue:** #9745

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
